lib/MusicBrainz/Server/Entity/URL/BookBrainz.pm

```diff
@@ -0,0 +1,19 @@
+package MusicBrainz::Server::Entity::URL::BookBrainz;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'BookBrainz' }
+
+__PACKAGE__->meta->make_immutable;
+no Moose;
+1;
+
+=head1 COPYRIGHT AND LICENSE
+Copyright (C) 2019 MetaBrainz Foundation
+This file is part of MusicBrainz, the open internet music database,
+and is licensed under the GPL version 2, or (at your option) any
+later version: http://www.gnu.org/licenses/gpl-2.0.txt
+=cut
```

Comment:
It is missing blank lines which delimit paragraphs, see template in [coding style](https://musicbrainz.org/doc/Development/Server_Coding_Style#Templates).